src/Home.js

```diff
@@ -26,13 +27,23 @@ function Home () {
       }),
   })
   
-  const allGirls = data && ('pages' in data) ? data.pages.flat() : []
+  const allGirls = data && ('pages' in data) ? data.pages.flat() : [];
+
+  const valueDir = (dir) => {
```

Comment:
As have been mentioned in the other comment, enumerating swiping direction is not usable to be sent as `value`. We can instead append the direction in the `action`

Reply:
noed